pubspec.yaml

```diff
@@ -18,17 +18,18 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  cookie_jar: ^3.0.1
-  device_info_plus: ^8.1.0
-  flutter_web_auth_2: ^2.0.4
-  http: ^0.13.5
-  package_info_plus: ^3.0.3
-  path_provider: ^2.0.13
-  web_socket_channel: ^2.3.0
-  universal_html: ^2.0.9
+  cookie_jar: ^4.0.3
+  device_info_plus: ^9.0.2
+  flutter_web_auth_2: ^2.1.4
+  http: '>=0.13.6 <1.0.1'
+  package_info_plus: ^4.0.2
+  path_provider: ^2.0.15
+  web_socket_channel: ^2.4.0
+  universal_html: ^2.2.2
```

Comment:
Possible breakign change for cookie_jar, device_info_plus, package_info_plus,

Reply:
working.